weldx/core.py

```diff
@@ -450,7 +467,7 @@ def _interp_time_discrete(self, time: Time) -> xr.DataArray:
         """Interpolate the time series if its data is composed of discrete values."""
         return ut.xr_interp_like(
             self._data,
-            {"time": time.as_timedelta()},
+            {"time": time.as_timedelta_index()},
```

Comment:
we should pass the reference time here as well

Reply:
well, if you use `as_data_array` here, you get the "non-monotonic" error I mentioned above 😉